recipes/argparse/all/conanfile.py

```diff
@@ -40,10 +40,12 @@ def source(self):
 
     def package(self):
         self.copy("LICENSE", src=self._source_subfolder, dst="licenses")
-        self.copy("*.hpp", src=os.path.join(self._source_subfolder, "include"), dst=os.path.join("include", "argparse"))
+        self.copy("*.hpp", src=os.path.join(self._source_subfolder, "include"), dst="include")
 
     def package_id(self):
         self.info.header_only()
 
     def package_info(self):
-        self.cpp_info.includedirs.append(os.path.join("include", "argparse"))
+        self.cpp_info.names["cmake_find_package"] = "argparse"
```

Comment:
Keep the include folder.
```suggestion
        self.cpp_info.includedirs.append(os.path.join("include", "argparse"))
        self.cpp_info.names["cmake_find_package"] = "argparse"
```